WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
Thanks for pointing out this super-scenario, comparison seems safer then =)
I've already tested it on the staging, it behaves the same way as when you register for a competition, what causes an email to be "sent" (it always breaks for staging thought).